sync/command.py

```diff
@@ -689,6 +695,10 @@ def do_landable(git_gecko,
         count = 0
         for pr, _, status in unlandable:
             count += 1
+            if blocked and status in (LandableStatus.ready,
+                                      LandableStatus.upstream,
+                                      LandableStatus.skip):
```

Comment:
for myself, 

`LandableStatus.ready` (and others) returns a unicode string
https://github.com/mozilla/wpt-sync/blob/1f74819e59cedfa05f1f38a3cd91237f8becf90c/sync/sync.py#L250-L267

ok sake, because it's part of a non unique tuple, but could have a edge effect if editer later to one value.

```python
>>> statuses = ('not_valid', 'weird', 'ok')
>>> status = 'valid'
>>> if status in statuses:
...     print('oopsie')
... 
>>> statuses = ('not_valid', None, None)
>>> if status in statuses:
...     print('oopsie')
... 
>>> statuses = ('not_valid')
>>> if status in statuses:
...     print('oopsie')
... 
oopsie
```

while

```python
>>> statuses = ['not_valid']
>>> if status in statuses:
...     print('oopsie')
... 
```


So will not fail in this context, but can be dangerous.

Reply:
I'm very confused by this comment :) The enum statuses aren't strings, they're instances of the enum subtype:
```
In [10]: type(LandableStatus.ready)                                                                                                                                                                                
Out[10]: <enum 'LandableStatus'>
```

We allow getting a string from those:
```
In [11]: LandableStatus.ready.reason_str()                                                                                                                                                                         
Out[11]: 'Ready'
```
But that's only really for display.